src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,63 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+
+        DriverService driverService = (DriverService)
+                injector.getInstance(DriverService.class);
+        Driver driverOne = new Driver("Shinyo Nakano", "NO467386");
+        driverService.create(driverOne);
```

Comment:
What the reason create it 2 times? It not a code mistake, but l guess it's better to do with other creations below